Snakefile

```diff
@@ -161,8 +160,9 @@ rule download_local_data:
 rule download_cancerhotspots:
     output: f'{DATA_DIR}/cancerhotspots/cancerhotspots.v2.maf'
     shell: dedent(f'''\
+        mkdir -p {DATA_DIR}/cancerhotspots
         cd {DATA_DIR}/cancerhotspots
-        wget http://download.cbioportal.org/cancerhotspots/cancerhotspots.v2.maf.gz
+        wget https://www.cancerhotspots.org/files/hotspots_v2.xls
```

Comment:
Is hotspots_v2.xls the file we want here? I don't think the loader can process it.

Reply:
Right, undoing this change. We may have to switch to it if we don't hear back from them though